Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -170,6 +147,29 @@ If not installing with Helm, here are the steps to install with manifests.
 
 2. Create [common-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/common-external.yaml) and [cluster-external.yaml](https://github.com/rook/rook/blob/master/deploy/examples/cluster-external.yaml)
 
+### Import the Source Data
+
+1. Paste the above output from `create-external-cluster-resources.py` into your current shell to allow importing the source data.
+
+1. The import script in the next step uses the current kubeconfig context by
+   default. If you want to specify the kubernetes cluster to use without
+   changing the current context, you can specify the cluster name by setting
+   the KUBECONTEXT environment variable.
+
+   ```console
+   export KUBECONTEXT=hub-cluster
```

Comment:
What is the hub-cluster? I might suggest to remove this example line. Setting an env var is common knowledge IMO.

Reply:
Mostly we use export in the import script, and thus the change to using KUBECONTEXT, is it fine to update in this PR?

Reply:
Updated